### DISCUSSION TITLE
Welcome to MichalOS Discussions!

### DISCUSSION THREAD

**Discussion by prochazkaml** [prochazkaml/MichalOS#1] [General]
Hi, Michal here!

If you have any questions about this project or have ideas which you would want to see in the next release, by all means, go ahead and post them here! 😁

**Comment by VendelinSlezak** [comment]
Hello, you can show your project on https://wiki.osdev.org/Projects

**Reply by prochazkaml** [reply]
Yeah, I probably won't do it with MichalOS in its current stage. I have big plans for this project (which will basically require to rewrite the entire OS & apps – not in assembly), but who knows if I ever actually get around to it.
